ebb_lint/test/test_flake8.py

```diff
@@ -1,13 +1,15 @@
+# -*- coding: utf-8; -*-
 from __future__ import unicode_literals
 
 import ast
 import functools
+import pytest
 import re
+import six
 import sys
 
-import pytest
-import six
 from flake8.engine import get_parser
+from io import BytesIO
```

Comment:
I suspect you and I are using different standards for "what order should imports be sorted in," and while mine's `*much handwaving*`, I bet yours is actually written down. Share the source of it with me so I can learn from it? :)